torch/csrc/jit/ir.cpp

```diff
@@ -449,19 +443,42 @@ void Graph::lint() const {
         }
       }
     }
+    JIT_ASSERT(anticipated_uses[n] == static_cast<int64_t>(n->inputs_.size()));
```

Comment:
it's an estimate of number of inputs, based on uses of previous nodes, so wouldn't it be clearer to call it `anticipated_inputs`?